src/platform/packages/shared/response-ops/alerts-table/components/alerts_table.tsx

```diff
@@ -470,7 +467,7 @@ const AlertsTableContent = typedForwardRef(
       () =>
         ({
           ...additionalContext,
-          columns,
+          columns: columnsWithFieldsData,
```

Comment:
I used this name to clarify that these columns sent to the underlying data grid are enriched with fields data coming from the fields API